### ISSUE TITLE
Special command \skip and \until no longer functional in ALIASES

### ISSUE THREAD

**Issue by thom-sch** [doxygen/doxygen#8015]
Using `\skip` and `\until` inside an `ALIASES` does not work in V.1.8.19 and 1.8.20 like this:
```
ALIASES          = "Snippet{2}=\skip \1 ^^ \until \2" \
                         "Snippet{1}=\Snippet{\1_start,\_end}"
```
**Screenshots**
s. attached zip-File

**To Reproduce**
- Define the 2 aliases in the configuration file
- Include some snippets with this these aliases:
  `\dontinclude modul1.cpp`

 This includes a snippet from line containing "example1_start" upto "example1_end"
 works always:
```
 \skip example1_start
 \until example1_end
```
 This should include a snippet from line containing "example2_start" upto "example2_end"
  works only in V.1.8.18:
```
  \Snippet{example2}
```
 This should include a snippet from line containing "example3_start" upto "example3_end"
  works only in V.1.8.18:
```
  \Snippet{example3_start,example3_end}
```
**Version**
Works fine in V.1.8.18
works not in V.1.8.19/20

** complete example s. attached ZIP-file **
[Test10_skip_until.zip](https://github.com/doxygen/doxygen/files/5183158/Test10_skip_until.zip)

Regards from Meerbusch
Thomas Schneider

**Comment by albert-github** [comment]
I've just pushed a proposed patch, pull request #8016

**Comment by albert-github** [comment]
Code has been integrated in master on github (please don't close the issue as this will be done at the moment of an official release).

**Comment by doxygen** [comment]
This issue was previously marked 'fixed but not released',
which means it should be fixed in doxygen version 1.9.0.
Please verify if this is indeed the case. Reopen the
issue if you think it is not fixed and please include any additional information
that you think can be relevant (preferably in the form of a self-contained example).
